todo_app/data/trello_items.py

```diff
@@ -0,0 +1,88 @@
+import os
+import requests
+from requests.api import request
```

Comment:
Super minor, but I dont think this import is used?

Reply:
Yep, I'll sort these out as part of the other exercise :D